rest_framework_serializer_extensions/serializers.py

```diff
@@ -1,20 +1,17 @@
-from __future__ import absolute_import
 from collections import OrderedDict
 from pprint import pformat
 
 from django.core.exceptions import FieldDoesNotExist
 from django.db.models import Prefetch
 from django.db.models.fields.related import ForeignKey
 from django.db.models.query import QuerySet
-from django.utils import six
 from rest_framework import serializers
 from rest_framework.fields import empty
 
 from rest_framework_serializer_extensions import (
     fields as custom_fields, utils
 )
 
-
```

Comment:
This looks like a flake8 regression?

Reply:
Hm, PyCharm takes out the second empty line when I "organize imports" -- I'll add it back in again